### PR TITLE
chore(wren-ai-service): langfuse analysis

### PR DESCRIPTION
**Pull request by cyyeh**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new `report-gen` command for generating reports based on Langfuse traces.
	- Added functionality for retrieving and filtering traces from the Langfuse API.

- **Removals**
	- Removed the `langfuse_dashboard` target from the Justfile.
	- Deleted the environment variable configuration for the Langfuse service.

- **Chores**
	- Cleaned up project configuration and tooling related to trace analysis and reporting.

### CODE REVIEW

File: wren-ai-service/eval/llm_trace_analysis/report_gen.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+    if only_errors:
+        return [
+            _get_output(trace, trace_name)
+            for trace in traces
+            if json.loads(trace.output).get("metadata", {}).get("error_type", "")
+        ]
+    else:
+        return [_get_output(trace, trace_name) for trace in traces]
+
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/eval/llm_trace_analysis/report_gen.py]: fixed

File: wren-ai-service/eval/llm_trace_analysis/report_gen.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+def get_langfuse_client() -> AsyncFernLangfuse:
+    public_key = os.getenv("LANGFUSE_PUBLIC_KEY")
+    secret_key = os.getenv("LANGFUSE_SECRET_KEY")
+    host = os.getenv("LANGFUSE_HOST")
+    return init_langfuse_api_client(
+        public_key=public_key,
+        secret_key=secret_key,
+        host=host,
+    )
+
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/eval/llm_trace_analysis/report_gen.py]: fixed

File: wren-ai-service/eval/llm_trace_analysis/report_gen.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    release: Optional[str] = None,
+) -> List[TraceWithDetails]:
+    # Get first page to determine total pages
+    first_page = await client.trace.list(
+        name=name,
+        page=1,
+        from_timestamp=from_timestamp,
+        to_timestamp=to_timestamp,
+        release=release,
+    )
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/eval/llm_trace_analysis/report_gen.py:59]: I don't need that at the moment

File: wren-ai-service/eval/llm_trace_analysis/report_gen.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    else:
+        return [_get_output(trace, trace_name) for trace in traces]
+
+
+def save_report(report_by_release: dict):
+    with open(
+        f"outputs/report_{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}.json", "w"
+    ) as f:
+        json.dump(report_by_release, f)
+
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/eval/llm_trace_analysis/report_gen.py]: fixed

**Review by paopa — APPROVED** [review]

**Pull request merged**